### ISSUE TITLE
Close button shows on a11y-push notify

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt-contrib-core#376]
### Subject of the issue
a11y-push notifications are for screen readers only, not visual UI should be shown.
Currently the template always shows the close button for push notifications.

https://github.com/adaptlearning/adapt-contrib-core/blob/c54cffec551a2311ab22c60b87a0a80274bf6aaa/templates/notifyPush.hbs#L24-L26
